source/transactions/tests/error-labels.yml

```diff
@@ -654,3 +654,165 @@ tests:
       collection:
         data:
           - _id: 1
```

Comment:
There is a a test for this "add unknown commit label to writeConcernError ShutdownInProgress" and the driver *should* add the UnknownTransactionCommitResult  label in these cases since those are retryable write errors.

Reply:
Yep, I got it backwards.  :|